plugin/health/README.md

```diff
@@ -33,8 +29,8 @@ health [ADDRESS] {
 * Where `lameduck` will make the process unhealthy then *wait* for **DURATION** before the process
   shuts down.
 
-If you have multiple Server Blocks and need to export health for each of the plugins, you must run
-health endpoints on different ports:
+If you have multiple Server Blocks and need to export health, you must run health endpoints on
```

Comment:
Would this still be the case if the health beck is being rolled up into a single check?